CC##99.CBL

```diff
@@ -589,7 +589,8 @@
            MOVE "Good" TO OUT-FIELD-3                                           
            .                                                                    
        5400-WRITE-OUTPUT-RECORD.                                                
-      *    WRITE OUTPUT-RECORD                                                  
+      *    WRITE                                                                
```

Comment:
I accidentally added this file while committing. This file is the output of the merged source and tests, thus it will be overwritten the next time Cobol-Check is run. I do not see a reason to change this. However I will make sure to ignore this file on future commits, as to not clutter the file changes.